docker/terraform/main.tf

```diff
@@ -1,6 +1,15 @@
 # Provider for the Root namespace
 provider "vault" {}
 
+# Log to stdout of the container
```

Comment:
Good thinking! 💪

Reply:
Thanks, we use this quite often now that more Vaults are deployed as containers..